dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -25,6 +24,31 @@ def update
     end
   end
 
+  def get_evaluation
+    learning_goal_evaluation = LearningGoalEvaluation.find_by(
+      user_id: learning_goal_evaluation_params[:user_id],
+      teacher_id: current_user.id,
+      learning_goal_id: learning_goal_evaluation_params[:learning_goal_id]
+    )
+
+    if learning_goal_evaluation&.teacher_id == current_user.id
+      render json: learning_goal_evaluation
+    else
+      head :not_found
+    end
+  end
+
+  def get_or_create_evaluation
+    learning_goal_evaluation = LearningGoalEvaluation.find_or_create_by(
```

Comment:
nit: use `find_or_create_by!` so that an error is thrown if this line fails ([doc link](https://apidock.com/rails/v4.0.2/ActiveRecord/Relation/find_or_create_by%21))

Reply:
done